src/pages/VenuePage/VenuePage.tsx

```diff
@@ -172,12 +170,12 @@ export const VenuePage: React.FC = () => {
   // @debt refactor how user location updates works here to encapsulate in a hook or similar?
   useEffect(() => {
     if (user && profile && venueId && venueTemplate) {
-      mixpanel.track("VenuePage loaded", {
+      window.mixpanel.track("VenuePage loaded", {
```

Comment:
can we create a mixpanel service/utils with all the tracking functions to have them all in the same place?

Reply:
fixed in 45c8facc226c9f6cd04226cd497c5d3662ab2527